politeiad/plugins/pi/pi.go

```diff
@@ -299,6 +299,9 @@ type ProposalMetadata struct {
 	StartDate int64  `json:"startdate"` // Start date, Unix time
 	EndDate   int64  `json:"enddate"`   // Estimated end date, Unix time
 	Domain    string `json:"domain"`    // Proposal domain
+
+	// TODO this is temporary and should be done in a separate PR.
+	LegacyToken string `json:"legacytoken"`
```

Comment:
We should break this out into a separate PR with validation that prevents this from being set during normal proposal submissions.

Reply:
💯

Reply:
Done in: https://github.com/decred/politeia/pull/1626, will ditch it from here once 1626 is merged.

Reply:
It's been merged.